marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-custom-reader.json

```diff
@@ -0,0 +1,19 @@
+{
+  "role-name": "hub-central-custom-reader",
+  "description": "Permits reading/reviewing custom steps in Hub Central",
```

Comment:
Since custom steps do not exist as a separate artifacts and only exist inline within flows, should "data-hub-flow-reader" role be added and roles "hub-central-load-reader", "hub-central-mapping-reader","data-hub-match-merge-reader" be dropped ?

Reply:
I think this story is coming earlier than it should be, but it doesn't hurt anything. But there's also not really a way to test it from a user's perspective.

Reply:
If access to viewing custom steps have to be restricted in DHFPROD-4997, DHFPROD-4998, then, "data-hub-flow-reader" role should be added right ?

Reply:
@srinathgit Similar to the other step types, the user should have access to the step document, not the flow. The primary information of the custom step is contained in the step definition.

Since all Custom step types will appear under the Custom tab, I believe you are correct that we can drop the hub-central-loader-reader, hub-central-mapping-reader, etc., roles.